### PR TITLE
adding image to header, adjusting css

### PR DESCRIPTION
**Pull request by karindalziel**

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -8,10 +8,12 @@
 </head>
 <body id="top-of-page" class="{{ layout.format }}">
 	{% unless page.skip_navigation %}
-	
+	<div class="title-surround">
```
Comment (karindalziel) [inline _layouts/default.html:11]: Added title-surround div to apply flex to head

File: _layouts/default.html [hunk 2]
```diff
@@ -8,10 +8,12 @@
...
+	  <img src="{{ '/assets/img/LogoPrint_trans.png' |relative_url }}" id="main-logo">
```
Comment (karindalziel) [inline _layouts/default.html:12]: added the larger version of the logo, that's the version on the live site

File: _layouts/page.html [hunk 3]
```diff
@@ -15,7 +15,7 @@
 	{% endunless %}
 	{% endif %}
 		
-	<div id="contents" class="large-8 small-12 columns {% if page.title == "Search" %} large-offset-2{% endif %}">
+	<div id="contents" class="large-8 small-12 columns {% if page.title == 'Search' %} large-offset-2{% endif %}">
```
Comment (karindalziel) [inline _layouts/page.html:18]: not sure if it matters, just saw some quotes within quotes here

File: _sass/main.scss [hunk 4]
```diff
@@ -8,12 +8,44 @@ $navfont: 'Roboto', sans-serif;
...
+  }
+}
+
 // Changing nav colors 
 
 .top-bar-section>ul>.divider,
 .top-bar-section>ul>[role="separator"],
 .top-bar, #navigation, .top-bar-section ul li, .top-bar-section ul, .dropdown li,
-.top-bar-section .dropdown li {
+.top-bar-section .dropdown li,
```
Comment (karindalziel) [inline _sass/main.scss:46]: getting rid of yellow in sub-sub dropdowns

**Comment by quinnanya** [comment]
CSS wizardry by Karin

**Pull request merged**